Userland/allocate.cpp

```diff
@@ -37,14 +37,14 @@ void usage(void)
     exit(1);
 }
 
-enum Unit { Bytes,
+enum class Unit { Bytes,
```

Comment:
These seem like completely arbitrary unrelated changes :)

Reply:
I changed this because of a naming collision with `using AK::Bytes = Span<u8>`.

https://github.com/SerenityOS/serenity/pull/2884/files#diff-d0f68934dcacd7f05ac0792071e08ad6R123-R124

Reply:
A-ha! Roger that. This looks very nice :)